addons/knowledge/models/knowledge_article.py

```diff
@@ -820,7 +852,7 @@ def _get_invite_url(self, partner):
 
     # TODO: remove me - for test purpose only
     def invite_test(self):
-        self.invite_member(access_rule='read', partner_id=self.env.ref('base.partner_demo_portal').id)
+        self.invite_members('read', self.env.ref('base.partner_demo_portal').id)
```

Comment:
well you ave to remove the .id otherwise it will crash, but we will delete this fonction anyway..